redhat/lyrionmusicserver.spec

```diff
@@ -94,6 +94,8 @@ Vendor:		Lyrion Community
 
 
 Requires:	perl >= 5.10.0
+PreReq:         /bin/bash /usr/bin/getent /usr/bin/cp /usr/bin/ln
+PreReq:         /usr/sbin/useradd /usr/sbin/groupadd /usr/bin/find
```

Comment:
I don't think the `PreReq` header has been used for a long time.  These days `Requires(pre)` (and `Requires(post)`) are used instead.

I generally prefer to have one dependency per line, sorted in alphabetical order, since it reduces Git merge request conflicts.

Reply:
Ok, I'll do that.